cypress/integration/slices/00-create.spec.js

```diff
@@ -1,53 +1,25 @@
-import path from "path";
-
 describe("Create Slices", () => {
   const sliceName = "TestSlice";
   const editedSliceName = "TestSlice2";
-  const lib = "slices--ecommerce"; // name of the first lib of the next project.
-  const pathToLib = path.join("e2e-projects", "next", "slices", "ecommerce");
-  const generatedPath = path.join(
```

Comment:
Probably keep these joins, as it makes it easier to add other OSs to the tests :)

Reply:
yes good point, will change

Reply:
Also, can we keep the e2e tests in the 2e tests directory?

Reply:
what do you mean?

Reply:
we already have a directory for e2e-projects, shouldn't we use it?

Reply:
ie: `e2e-projects/next-app`

Reply:
ah okay, can move it there 👍🏼

Reply:
thanks :)